samples/client/petstore/php/SwaggerClient-php/lib/ObjectSerializer.php

```diff
@@ -42,23 +42,26 @@ class ObjectSerializer
     /**
      * Serialize data
      *
-     * @param mixed $data the data to serialize
+     * @param mixed  $data   the data to serialize
+     * @param string $type   the SwaggerType of the data
+     * @param string $format the format of the Swagger type of the data
      *
      * @return string|object serialized form of $data
      */
-    public static function sanitizeForSerialization($data)
+    public static function sanitizeForSerialization($data, $type = null, $format = null)
```

Comment:
Yes, you are right.

I think I though that to be generally useful it would not make sense to provide only the format since the format is "just" a property of the type. Maybe I'm planning too much for the future.